breathecode/feedback/migrations/0024_survey_response_rate.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.9 on 2022-02-04 21:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Remove the two files related to the migration `breathecode/feedback/migrations/0024_survey_response_rate.py` and `breathecode/feedback/migrations/0025_alter_survey_response_rate.py` and run the migrations again to create the migrations just in one file